### ISSUE TITLE
docs on usage are out of sync and fail due to API changs in v2

### ISSUE THREAD

**Issue by joinr** [clj-python/libpython-clj#173]
https://clj-python.github.io/libpython-clj/Usage.html

```clojure
(require '[libpython-clj.python
                 :refer [as-python as-jvm
                         ->python ->jvm
                         get-attr call-attr call-attr-kw
                         get-item att-type-map
                         call call-kw initialize!
                         as-numpy as-tensor ->numpy
                         run-simple-string
                         add-module module-dict
                         import-module
                         python-type]])
```
fails.

Removing call, as-numpy, as-tensor, ->numpy and changing namespace works:

```clojure
(require '[libpython-clj2.python
                 :refer [as-python as-jvm
                         ->python ->jvm
                         get-attr call-attr call-attr-kw
                         get-item
                         initialize!
                         run-simple-string
                         add-module module-dict
                         import-module
                         python-type]])
```

I haven't checked any other examples.

**Comment by behrica** [comment]
I take a look at this, with  the aim to fix it

**Comment by behrica** [comment]
I fixed them all with d37f06c48b75365f5dcb0dc2896cc9c3daabf89a


and propose to close the issue.

**Comment by cnuernber** [comment]
Closed.  I am sure there are more places but this is fine for now.
